.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-.env.*
\ No newline at end of file
+.env.*
+.env.tes
+.env.development
```

Comment:
`.env.*` will be fine here. No need for the separate ones for development and test